sonar-cxx-plugin/src/test/java/org/sonar/plugins/cxx/TestUtils.java

```diff
@@ -171,16 +172,16 @@ public static void addInputFile(DefaultFileSystem fs,
         temp.setCharAt(0, Character.toUpperCase(temp.charAt(0)));
         path = temp.toString();
       }
-      inputFile = new DefaultInputFile(path);
-      inputFile.setAbsolutePath(path);
+      inputFile = new DefaultInputFile("key", path);
```

Comment:
> and what about if we just mock everything related with filesystem

In the past we had a lot of issues with differences on Windows and Linux file system. Not sure if it is a good idea to mock this all away?